--- /etc/influxdb/influxdb.conf.org	2016-11-03 15:42:21.943348924 +0100
+++ /etc/influxdb/influxdb.conf	2016-11-03 15:43:39.163346144 +0100
@@ -4,7 +4,7 @@
 # The data includes a random ID, os, arch, version, the number of series and other
 # usage data. No data from user databases is ever transmitted.
 # Change this option to true to disable reporting.
-reporting-disabled = false
+reporting-disabled = true
 
 # we'll try to get the hostname automatically, but if it the os returns something
 # that isn't resolvable by other servers in the cluster, use this option to
@@ -194,11 +194,11 @@
 ###
 
 [[graphite]]
-  enabled = false
-  # database = "graphite"
-  # bind-address = ":2003"
-  # protocol = "tcp"
-  # consistency-level = "one"
+  enabled = true
+  database = "graphite"
+  bind-address = ":2003"
+  protocol = "tcp"
+  consistency-level = "one"
 
   # These next lines control how batching works. You should have this enabled
   # otherwise you could get dropped metrics or poor performance. Batching
@@ -225,6 +225,12 @@
   #   # Default template
   #   "server.*",
   # ]
+  templates = [
+    "fastnetmon.hosts.* app.measurement.cidr.direction.function.resource",
+    "fastnetmon.networks.* app.measurement.cidr.direction.resource",
+    "fastnetmon.total.* app.measurement.direction.resource"
+  ]
+
 
 ###
 ### [collectd]
